explanation/cli_tools.rst

```diff
@@ -0,0 +1,71 @@
+Understanding juju vs jaas vs jimmctl
+=====================================
+
+This document aims to explain the differences between the various CLI tools you may encounter when using Juju and JAAS.
+
+=======
+Summary
+=======
+
+.. list-table::
+   :widths: 25 25 25 25
+   :header-rows: 1
+
+   * - 
+     - ``juju``
+     - ``jaas``
+     - ``jimmctl``
+   * - Purpose
```

Comment:
Currently the headers are captialized and the table looks like this:

![cli-tools](https://github.com/canonical/jaas-documentation/assets/36728931/0fb6499c-0526-409c-ba20-5468f244b9ea)

Can't we de-capitalize the header row? I'm not that familiar with RST.

Reply:
Also, I think the installation link texts should be surrounded by double backticks.

Reply:
I kinda like them capitalised aha

Reply:
I've changed `:header-rows: 0`, now the first row is not capitalised. I think it looks better, more in line with the fact that we are referencing the CLI tools.